lib/widgets/poll.dart

```diff
@@ -73,10 +74,14 @@ class _PollWidgetState extends State<PollWidget> {
         textBaseline: localizedTextBaseline(context),
         children: [
           ConstrainedBox(
-            constraints: const BoxConstraints(minWidth: 25 + 5).tighten(height: 25 + 5),
+            constraints: const BoxConstraints(
+              minWidth: 39 + 5).tighten(height: 39 + verticalPadding * 2),
             child: Padding(
-              padding: const EdgeInsetsDirectional.only(bottom: 5, end: 5),
+              padding: const EdgeInsetsDirectional.only(
+                end: 5, top: verticalPadding, bottom: verticalPadding),
               child: Container(
+                // This is only in effect
+                // when the vote count has more than 2 digits.
```

Comment:
I'm a little suspicious of a comment like this :) because I don't know if it's true as written for all values of the user's system bold-text and text-size settings. If this needs a comment, maybe it could be something like:

```dart
                  // Inner padding preserves whitespace even when the text's
                  // width approaches the button's min-width (e.g. because 
                  // there are three digits).
```